source/cursorManager.py

```diff
@@ -42,8 +42,11 @@ def __init__(self, parent, cursorManager, text, caseSensitivity):
 		# Translators: Dialog text for NvDA's find command.
 		textToFind = wx.StaticText(self, wx.ID_ANY, label=_("Type the text you wish to find"))
 		findSizer.Add(textToFind)
-		self.findTextField = wx.TextCtrl(self, wx.ID_ANY)
-		self.findTextField.SetValue(text)
+		self.findTextField = wx.ComboBox(self, wx.ID_ANY, choices = searchEntries,style=wx.CB_DROPDOWN)

```

Comment:
I think it might be best to add combo box as something that could benefit from GUI helper layout enhancements. But for now, it should look okay on screen.

Reply:
My understanding is that an edit and a closed combo should occupy the same space, so I also consider this is ok for now.

Reply:
I have checked the find dialog visually. It seems to be quite similar to how it was already. However, it seems that the find dialog was missed when guiHelper was created. The spacing and borders are not so good. I don't suggest this gets updated with this PR. Though I would love to see a PR that took it on.

Reply:
Agreed. This should be addressed on a PR latter on .... not being sighted I may try to fix that but I will need help to validate. Never the less this should not prevent this PR from being merged.